charts/corda-dev-prereqs/templates/kafka/kafka-statefulset.yaml

```diff
@@ -107,6 +111,15 @@ spec:
                 path: "tls.key"
               - key: "ca.crt"
                 path: "ca.crt"
-        - name: "kafka-data"
-          persistentVolumeClaim:
-            claimName: {{ include "corda-dev-prereqs.kafkaName" . | quote }}
\ No newline at end of file
+  volumeClaimTemplates:
+    - metadata:
+        name: "kafka-data"
+        labels:
+          {{- include "corda-dev-prereqs.labels" . | nindent 10 }}
+      spec:
+        accessModes:
+          - "ReadWriteOnce"
+        resources:
+          requests:
+            storage: "500M"
```

Comment:
Will that be enough?

Reply:
It was sufficient for me to at least start a Corda cluster against it but only time will tell! Given we weren't writing anything to it, the size of the previous PVC doesn't help. This is only intended for local development use.